src/app/profile/page.tsx

```
@@ -1,201 +1,360 @@
-            thumbnail: '/post/post-example.png',
-            userProfile: '/profile/profile.svg',
-        },
-        {
-            id: 6,
-            user: '해ㅠ피',
-            title: '해피해피하숙',
-            region: '강서구',
-            price: '30',
-            thumbnail: '/post/post-example.png',
-            userProfile: '/profile/profile.svg',
+  const { data: postData, loading: postLoading, error: postError, fetchMore } = useQuery<UserPostListResponse>(
```

ESLint

[error] 'postData' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)